pocs/scheduler/constraint.py

```diff
@@ -159,3 +120,40 @@ def get_score(self, time, observer, observation, **kwargs):
 
     def __str__(self):
         return "Moon Avoidance"
+
+
+class Altitude(BaseConstraint):
```

Comment:
The move down the file seems unnecessary, FWIW.

Reply:
This is a direct pep8 change: https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
No, I mean the order of the classes was changed, with Altitude moved down the file. It was above class Duration, now is below.

Reply:
Oh, got it. Yes, this was originally added to the bottom of the file as a new constraint named `Horizon`, but then I realized after writing it that it basically replaced the existing Altitude constraint (which wasn't being used at all) so I deleted the one  from above and renamed the new one I had written.

Reply:
Alphabetical order probably makes the most sense.